assets/src/scripts/web-services/groundwater-levels.js

```diff
@@ -7,18 +7,25 @@ import config from 'ui/config';
  *  Formats a URL for the purpose of downloading groundwater levels data in RDB format
  * @param  {String}  siteno - A code that uniquely identifies a monitoring location
  * @param  {String}  parameterCode - USGS five digit parameter code
- * @param {String} startTimeDT - ISO 8601 time
- * @param {String} endDT- ISO 8601 time
+ * @param {String} startTime - ISO 8601 time
+ * @param {String} endTime- ISO 8601 time
  * @param {String} format - The format of the data returned from the water services API, such as 'json' or 'rdb'
  * @return {String} The URL used to contact waterservices.usgs.gov
  */
-export const getServiceURLSGroundwater = function({siteno, parameterCode, period= null, startDT=null, endDT=null, format}) {
+export const getGroundwaterServiceURL = function ({
```

Comment:
Also changed the name of the get*URL functions to follow the same pattern as above.